### PR TITLE
Rewrite cruftfiles postbox

### PR DESCRIPTION
**Pull request by Moppa5**
#### What are the main changes in this PR?
Rewrite the cruftfiles postbox with the new design.

- Autoloadable CruftRemover PHP class
- Common JS file for all cruft remover widgets
- Some css style changes & additions
- Move cruft files listing under `seravo-container` so that it overflows rather than consuming the whole page

#### Manual testing steps?
One can use for example the following bash script for generating some mock cruft files when testing.
```
#!/bin/bash
for ((index = 0; index <32; index++))
do
touch /data/wordpress/$index.sql
done
```

### CODE REVIEW

File: js/cruftremover.js [hunk 1]
```diff
@@ -0,0 +1,172 @@
...
+          }
+        );
+      }
+    }
+  );
+}
+
+/**
+ * Remove the given cruft entries on a section with animation.
+ * @param {array} remove_list List of the content to remove.
```
Comment (JoosuaKoskinen) [inline js/cruftremover.js:62]: \@Param comments, awesome!

File: src/modules/pages/security.php [hunk 2]
```diff
@@ -391,24 +409,68 @@ public static function get_last_successful_logins() {
...
-      <div id="cruftfiles_status_loading">
-        <?php _e('Searching for files...', 'seravo'); ?>
-        <img src="/wp-admin/images/spinner.gif">
-      </div>
-    </div>
-    </p>
-    <?php
+  /**
+   * Build func for cruft files postbox.
+   * @param $base The base of the postbox to add elements.
```
Comment (JoosuaKoskinen) [inline src/modules/pages/security.php:414]: The `$base` is missing type.

File: style/cruftremover.css [hunk 3]
```diff
@@ -15,6 +15,19 @@
   margin-bottom: 20px;
 }
 
+.cruft-delete-button {
```
Comment (JoosuaKoskinen) [inline style/cruftremover.css:18]: Why doesn't it use `.button`?

File: src/modules/pages/security.php [hunk 4]
```diff
@@ -2,6 +2,7 @@
 
 namespace Seravo;
 
+use Seravo\Ajax\AjaxHandler;
```
Comment (JoosuaKoskinen) [inline src/modules/pages/security.php:5]: Please make it `\Seravo\...`

**Review by JoosuaKoskinen — CHANGES_REQUESTED** [review]
The code is good! Just a couple of design notes: 

- The button looks different than other buttons in the plugin
cruftfile1
- The postbox is really tall, could be much shorter as there's the scrollbar
- There's a weird line at bottom right corner:
cruftfile2
- Maybe use WordPress popup like this instead of `confirm`?
cruftfile3

File: style/cruftremover.css (hunk 3 above)
Comment (Moppa5) [reply style/cruftremover.css:18]: Good point. Maybe too much reverse engineering and didn't even think about that. I will try to fit the consistent button as other widgets have as well :)

File: js/cruftremover.js [hunk 5]
```diff
@@ -0,0 +1,193 @@
...
+    // Init checkbox functionality
+    init_cruft_selects(section);
+
+    section.find('.button-primary').click(
+      function () {
+        //event.preventDefault();
+        /*var is_user_sure = confirm(cruftremover_l10n.confirm);
+        if (! is_user_sure) {
+          return;
+        } */
```
Comment (Moppa5) [inline js/cruftremover.js:137]: Should remove these...

**Review by JoosuaKoskinen — CHANGES_REQUESTED** [review]
- The "The selected files have..." is on bit weird place, almost touching the table. Is the text even needed? We could show the "Not removed" text on failure but on success customer will see the entry disappear from the table.
cruftfiles
- The weird line from previous review is still there

LGTM otherwise

**Pull request merged**